File: Algorithms/Compare_the_Triplets/compare_the_triplets.py
# Compare-the-Triplets

def CompareTheTriplets(a, b):
    aliceInitialPoint = 0
    bobInitialPoint = 0
    if len(a) == len(b):
        i = 0
        while i<len(a):
            if a[i] > b[i]:
                aliceInitialPoint+=1
            if a[i] < b[i]:
                bobInitialPoint+=1
            i=i+1
    else:
        print ('Data are Not same')

    print (aliceInitialPoint)
    print (bobInitialPoint)
    # return (aliceInitialPoint, bobInitialPoint)


alice = [3,4,99]
bob = [33,32,22]

CompareTheTriplets(alice, bob)

File: Algorithms/Compare_the_Triplets/compare_the_triplets.js
// Compare-the-Triplets

CompareTheTriplets=(a, b)=>{
    aliceInitialPoint = 0
    bobInitialPoint = 0

    if (a.length === b.length) {
        for (let i = 0; i < a.length; i++) {
            const element = a[i];
            if (a[i] > b[i]) {
                aliceInitialPoint++
            }
            if (a[i] < b[i]){
                bobInitialPoint++
            }
        }
    }
    else{
        console.log('Data are not same')
    }

    console.log(aliceInitialPoint, bobInitialPoint)
    // return (aliceInitialPoint, bobInitialPoint)
}


const alice = [17, 28, 30]
const bob = [99, 16, 8]

CompareTheTriplets(alice, bob)